programs/sunrise_beam/src/instructions/update_allocations.rs

```diff
@@ -0,0 +1,22 @@
+use anchor_lang::prelude::*;
+
+use crate::state::AllocationUpdate;
+use crate::UpdateBeamAllocations;
+
+pub fn handler(ctx: Context<UpdateBeamAllocations>, values: Vec<AllocationUpdate>) -> Result<()> {
+    let state = &mut ctx.accounts.state;
+
+    for value in values {
```

Comment:
what is meant by "values" here?

Reply:
I should probably name it `new_allocations`